Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/preferences/EncryptedPreferencesMigration.kt

```diff
@@ -0,0 +1,22 @@
+package de.rki.coronawarnapp.storage.preferences
+
+import de.rki.coronawarnapp.util.di.EncryptedPreferences
+import de.rki.coronawarnapp.util.di.Preferences
+import timber.log.Timber
+import javax.inject.Inject
+
+class EncryptedPreferencesMigration @Inject constructor(
+    @Preferences private val settingsPreferences: SettingsPreferences,
+    @EncryptedPreferences private val encryptedSettingsPreferences: SettingsPreferences
+) {
+
+    fun migrate() {
+        Timber.d("EncryptedPreferencesMigration START")
+        if (encryptedSettingsPreferences.containsData()) {
+            settingsPreferences.isNotificationsRiskEnabled = encryptedSettingsPreferences.isNotificationsRiskEnabled
```

Comment:
Do we need to do this one by one? Have you evaluated just taking the whole preference key/value map and moving  it from one encrypted to non encrypted preferences?

@kolyaopahle  @jurajkusnier